src/lib/apiErrorHandler.js

```diff
@@ -192,12 +195,9 @@ export const handleWorkerError = async (job, error, logContext) => {
     label = 'UNEXPECTED ERROR';
   }
 
-  if (error instanceof Sequelize.ConnectionError
-    || error instanceof Sequelize.ConnectionAcquireTimeoutError) {
-    const pool = sequelize?.connectionManager?.pool;
-    const usedConnections = pool ? pool.used.length : null;
-    const waitingConnections = pool ? pool.pending.length : null;
-    logger.error(`${logContext.namespace} Connection Pool: Used Connections - ${usedConnections}, Waiting Connections - ${waitingConnections}`);
+  if (error instanceof Sequelize.ConnectionAcquireTimeoutError) {
+    logger.error(`${logContext.namespace} - Critical error: Restarting server.`);
```

Comment:
Please make the error message state that this is the Sequelize.ConnectionAcquireTimeoutError that is the critical error